packages/protocol/contracts-0.8/stability/ExampleSixDecimalFeeCurrency.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.7 <0.8.20;
+
+import "@openzeppelin/contracts8/access/Ownable.sol";
+import "@openzeppelin/contracts8/token/ERC20/ERC20.sol";
+import "@celo-contracts/common/Initializable.sol";
+import "@celo-contracts-8/stability/interfaces/IFeeCurrency.sol";
+
+contract ExampleSixDecimalFeeCurrency is IFeeCurrency, ERC20, Ownable, Initializable {
```

Comment:
Note-to-self: 

The implementation is flawed, I think I have to either extend ERC20 and use the base constructor, or implement IERC20 and use the initializer.

My hunch is that I'll simplify, by removing Ownable and Initializable, and instead use the constructor.

In `Migrations.s.sol` I'll have to deploy the code differently, since the helper function deploys a proxied contract.